app/assets/v2/js/pages/new_bounty.js

```diff
@@ -131,45 +134,26 @@ $(document).ready(function() {
     } else {
       $('#auto_approve_workers_container').hide();
     }
-  });
-
-  $('input[name=amount]').on('change', function() {
-    const amount = $('input[name=amount]').val();
 
-    $('#summary-bounty-amount').html(amount);
-    $('#summary-fee-amount').html((amount / FEE_PERCENTAGE).toFixed(4));
-    populateBountyTotal();
+    if (val === 'reserved') {
+      $('#reservedForDiv').show();
+    } else {
+      $('#reservedForDiv').hide();
+    }
   });
 
-  $('select[name=denomination]').change(function(e) {
-    const token = tokenAddressToDetails(e.target.value).name;
+  var option = gettext('Reserved For - I will select a Gitcoin user.');
 
-    $('#summary-bounty-token').html(token);
-    $('#summary-fee-token').html(token);
-    populateBountyTotal();
-  });
+  $('select[name=project_type]').on('change', function() {
```

Comment:
❤️

Reply:
Hey @thelostone-mc . Didn't see these conflicts. How do we work them out?